book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -1,195 +1,487 @@
+* `threshold`: threshold for Gelman-Rubin convergence diagnostic (MGPRF); default is 1.2.
 
-## Brown Dog Configuration
+This information is currently used by the following PEcAn workflow function:
 
-This section describes how to connect to [Brown Dog](http://browndog.ncsa.illinois.edu). This will allow for conversions of data (currently only met data).
+- `PEcAn.MA::run.meta.analysis` - ???
 
-#### functions: 
+### `model`: Model configuration {#xml-model}
 
-`met.process()`
+This section describes which model PEcAn should run and some instructions for how to run it.
 
 ```xml
-<browndog>
-	<url>...</url>
-	<username>...</username>
-	<password>...</password>
-</browndog>
+<model>
+	<id>7</id>
+	<type>ED2</type>
+	<binary>/usr/local/bin/ed2.r82</binary>
+	<job.sh>module load hdf5</job.sh>
+	<config.header>
+		<!--...xml code passed directly to config file...-->
+	</config.header>
+</model>
 ```
 
-* **url** : [required] endpoint for Brown Dog to be used.
-* **username** : [optional] username to be used with the endpoint for Brown Dog.
-* **password** : [optional] password to be used with the endpoint for Brown Dog.
+Some important tags are as follows:
 
+- `id` -- The unique numeric ID of the model in the Bety `models` table. If this is present, then `type` and `binary` are optional since they can be determined from Bety.
+- `type` -- The model "type", matching the Bety `modeltypes` table `name` column. This also refers to which PEcAn model-specific package will be used. In PEcAn, a "model" refers to a specific version (e.g. release, git commit) of a specific model, and "model type" is used to link different releases of the same model. Model "types" also have specific PFT definitions and other requirements associated with them (e.g. the ED2 model "type" requires a global land cover database).
+- `binary` -- The file path to the model executable. If omitted, PEcAn will use whatever path is registered in Bety for the current machine. 
+- `job.sh` -- Additional options added to the `job.sh` script, which is used to execute the model. This is useful for setting specific environment variables, load modules, etc.
 
-## PFT Selection
+This information is currently used by the following PEcAn workflow function:
 
-The PEcAn system requires at least 1 PFT (Plant Functional Type) to be specified inside the `<pfts>` section. 
+`write.configs()`, `run.models()`
 
-#### functions:  
+#### Model-specific configuration {#xml-model-specific}
 
-`get.traits()`
+See the following:
 
+* [ED2][ED2 Configuration]
+* [SIPNET][SIPNET Configuration]
+* [BIOCRO][BioCro Configuration]
 
-#### tags
+#### ED2 specific tags {#xml-ed}
 
+Following variables are ED specific and are used in the [ED2 Configuration](ED2-Configuration).
+
+Starting at 1.3.7 the tags for inputs have moved to `<run><inputs>`. This includes, veg, soil, psscss, inputs.
```

Comment:
I thought we'd agreed in another thread that because the documentation is now versioned that we can drop long-deprecated features like this (probably does make sense though to discuss very recent changes)